packages/wallets/.eslintrc.js

```diff
@@ -1,11 +1,22 @@
 module.exports = {
-    root: true,
     extends: ['../../.eslintrc.js', 'eslint:recommended', 'plugin:react/recommended'],
-    parserOptions: { sourceType: 'module' },
     env: { es6: true },
+    overrides: [
```

Comment:
any particular reason for adding that exclusion?

Reply:
Allowing the code in those files to use variables that are not explicitly defined without triggering ESLint errors.